tests/integration/star_wars/test_errors.py

```diff
@@ -41,4 +41,18 @@ def test_skip_question_errors(self):
 
         # Then errors exists on page
         self.assertStatusOK()
-        self.assertInPage('This page has 1 errors')
+        self.assertInPage('This page has an error')
+        self.assertInPage('This <strong>must be corrected</strong> to continue.')
```

Comment:
It may be worth adding a test for multiple errors showing "These" as well.

Reply:
@dwyeradam This is covered below see line 58